src/node_http_parser.cc

```diff
@@ -74,6 +74,16 @@ const uint32_t kOnHeadersComplete = 1;
 const uint32_t kOnBody = 2;
 const uint32_t kOnMessageComplete = 3;
 
+const uint32_t kHeaders = 0;
+const uint32_t kUrl = 1;
+const uint32_t kVersionMajor = 2;
+const uint32_t kVersionMinor = 3;
+const uint32_t kMethod = 4;
+const uint32_t kStatusCode = 5;
+const uint32_t kStatusMessage = 6;
+const uint32_t kUpgrade = 7;
+const uint32_t kShouldKeepAlive = 8;
+const uint32_t kMessageInfoLength = 9;
```

Comment:
Maybe turn this into an enum?  If nothing else, they should probably be `static const` (so should the existing ones.)